src/dataValidation/validator.js

```diff
@@ -0,0 +1,32 @@
+const pattern = new RegExp(/^[a-zA-Z_ ]+$/);
+
+export const nameCheck = (name) => {
+  let nameResult = pattern.test(name);
+  if (!nameResult || name.length < 2) {
+    return false;
+  } else {
+    return true;
+  }
+};
+
+export const ageCheck = (age) => {
+  if (parseInt(age) >= 0) {
```

Comment:
`parseInt(age) >= 0` is already a boolean value, you can return it directly: 
`
ageCheck = (age) => {
 return parseInt(age) >= 0
}
`